packages/common/src/lib/entity/entity-table/entity-table.component.ts

```diff
@@ -185,6 +199,8 @@ export class EntityTableComponent implements OnInit, OnChanges, OnDestroy {
    */
   dataSource = new MatTableDataSource<object>();
```

Comment:
Quelle est la différence entre le dataSource et le processedRowData?
On pourrait typer le data du DataSource
```suggestion
  dataSource = new MatTableDataSource<RowData>();
```